src/components/Navbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'BOOKS',
+    },
+    {
+      id: 2,
+      path: '/categories',
+      text: 'CATEGORIES',
+    },
+  ];
+
+  return (
+    <nav className="navBar">
+      <ul>
+        <h1>BookStore CMS</h1>
+        {links.map((link) => (
+          <li key={link.id}>
+            <NavLink to={link.path} activeClassName="active-link" exact>
+              {link.text}
+            </NavLink>
+          </li>
+        ))}
+      </ul>
+      <p>icons</p>
```

Comment:
- [ ] This text appears even when I change from books to categories, please remove it from the categories view because as far as I understood it will be the placeholder of every book icon? If it is the category Icon then ignore this requirement.

Reply:
Thank you @e71az I removed the icon text